quarkus/tests/integration/src/test/java/org/keycloak/it/cli/dist/BuildCommandDistTest.java

```diff
@@ -76,14 +76,14 @@ void testFailRuntimeOptions(LaunchResult result) {
     @Launch({ "build" })
     void testFailInvalidOptionInEnv(LaunchResult result) {
         CLIResult cliResult = (CLIResult) result;
-        cliResult.assertError("Invalid value for option 'kc.db': invalid. Expected values are: dev-file, dev-mem, mariadb, mssql, mysql, oracle, postgres. From ConfigSource KcEnvVarConfigSource");
+        cliResult.assertError("Invalid value for option 'KC_DB': invalid. Expected values are: dev-file, dev-mem, mariadb, mssql, mysql, oracle, postgres");
     }
 
     @Test
     @RawDistOnly(reason = "Raw is enough and we avoid issues with including custom conf file in the container")
     public void testFailInvalidOptionInConf(KeycloakDistribution distribution) {
         CLIResult cliResult = distribution.run(CONFIG_FILE_LONG_NAME + "=" + Paths.get("src/test/resources/BuildCommandDistTest/keycloak.conf").toAbsolutePath().normalize(), "build");
-        cliResult.assertError("Invalid value for option 'kc.db': foo. Expected values are: dev-file, dev-mem, mariadb, mssql, mysql, oracle, postgres. From ConfigSource PropertiesConfigSource[source");
+        cliResult.assertError("Invalid value for option 'kc.db' in configuration file: foo. Expected values are: dev-file, dev-mem, mariadb, mssql, mysql, oracle, postgres");
```

Comment:
It'd be great to know what exactly is the configuration file. Is it feasible? WDYT?

Reply:
Possibly. With the way the logic is currently there can be only one configuration file - keycloak.conf or whatever the user provides to --config-file. It seemed good enough to refer to it abstractly.

Alternatively we can try to extract the short name, or even the full path (but I wasn't sure we even wanted full file paths in these messages).

Reply:
It will now show the filename instead.

Reply:
nice, thanks!